docs/learn/learn-extrinsics.md

```diff
@@ -0,0 +1,152 @@
+metadata is different (you will notice that on the Signer app the update takes longer for example).
+Having outdated metadata on the Signer app will prevent you from signing while on the extension you
+will be able to click the sign button but the extrinsic will likely fail similarly of having an
+outdated Ledger app. In general, failing to update metadata will most likely result in you not being
+able to transfer funds.
+
+## Verifying Extrinsics
+
+:::info Walk-through Video Tutorial
+
+Visit the
+[dedicated support page](https://support.polkadot.network/support/solutions/articles/65000179161-how-can-i-verify-what-extrinsic-i-m-signing-)
+and see [this technical explainer](https://youtu.be/bxMs-9fBtFk) tutorial to learn about how to
+verify extrinsics before signing them. The video will also mention potential attacks that can happen
+to you while signing for transactions. **Do not sign a transaction if you can't verify what you are
+signing or you suspect you might be signing a different extrinsic than the one intended.**
+
+:::
+
+Verifying the extrinsic you are signing can take some more time before signing for a transaction but
+it allows you to add an extra security step. There are a multitude of possible attacks that will
+prevent you to send funds to the desired destination account (see below).
+
+## How do Attacks look like
+
+In general, an attacker would make you think you are signing an extrinsic A when you are signing an
+extrinsic B. An Attack might come from:
+
+### Clipboard Memory
+
+:::info
+
+For a more detailed read about clipboard memory attacks see
+[this article](https://www.kaspersky.com/blog/cryptoshuffler-bitcoin-stealer/19976/).
+
+:::
+
+This is a common attack. The clipboard memory is that memory on you computer dedicated to copy-paste
+operations. There is malicious software that can be remotely installed on your computer and that can
+detect when a cryptocurrency address is copied. For example, you want to send funds to Address A
+(belonging to you) but after copying address A a malicious software swaps that address with Address
+B (belonging to an attacker). If you do not check the receiver address before signing the funds will
+be lost.
+
+### Corrupted Website
+
+This is a common attack that can happen if your extension is genuine, but the website (dApp) it is
+connected with is corrupted. In this scenario you want to perform Extrinsic A on the website, which
+will send Extrinsic B to the extension for signature. In this case the extension will show Extrinsic
+B. If you are using a Ledger device you have a second layer of verification, is it will also display
+Extrinsic B. In this case it is more difficult to spot the attack and it is up to the user to know
+the extrinsic that was created on the website in the first place.
+
+### Corrupted Browser Extension
+
+There is a minor risk that the browser extension such as the
```

Comment:
This scenario can happen if you have downloaded a malicious extension or a trusted extension, like the Polkadot-JS Browser extension, from a non-trusted source.